Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -62,7 +71,10 @@ static bool LoadConfig()
             return initialized;
         }
 
-
+        /// <summary>
+        /// Checks if an active session exists.
+        /// </summary>
```

Comment:
I applaud the diligence in summarizing private methods ^^

Reply:
Haha, not intentional, but doesn't hurt, so I'll keep it. We might need it in the future 😛